src/main/java/frc/robot/RobotContainer.java

```diff
@@ -129,9 +117,68 @@ public RobotContainer(BiConsumer<Runnable, Double> addPeriodic) {
 
     this.addPeriodic = addPeriodic;
 
-    configureBindings();
+    if (RobotController.getSerialNumber().equals("032414F0")) {
+      whichRobot = Robot.COMP;
+    } else if (RobotController.getSerialNumber().equals("0323CA18")) {
+      whichRobot = Robot.DEV;
+    } else if (RobotController.getSerialNumber().equals("03223849")) {
+      whichRobot = Robot.WALLE;
+    }
```

Comment:
Add an `else` statement that logs or alerts, with the serial number.  So we can quickly identify when a new robot-rio is being used.